app/src/components/v-input/v-input.vue

```diff
@@ -281,7 +281,9 @@ body {
 		flex-grow: 1;
 		align-items: center;
 		height: 100%;
-		padding: 0 var(--input-padding);
+		padding: var(--input-padding);
+		padding-top: 0px;
+		padding-bottom: 0px;
```

Comment:
How does this fix it? Isn't `padding: 0 var(--input-padding);` the same as

```css
padding-top: 0;
padding-right: var(--input-padding);
padding-bottom: 0;
padding-left: var(--input-padding);
```

Reply:
Haha, that's what you think until you notice that `--input-padding` is `12px 12px 12px 12px` so adding a 0 before it wont work.

Reply:
https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif

Only within the context of the color interface though! Maybe we should fix it just in there?


<img width="493" alt="CleanShot 2021-02-19 at 13 45 18@2x" src="https://user-images.githubusercontent.com/9141017/108547719-dd765200-72b8-11eb-8f43-12c99f8c3b28.png">

Reply:
I would say how it is is the right way to go, otherwhise we would have to do something like `.v-input .input {padding-left: 4px}` which would defeat the purpose of having `--input-padding` in the first place.

Reply:
I disagree. The --input-padding sets the default, but only the color interface has to override the padding-left for it's particular use case. We don't use that anywhere else, so i see it as an exception